### DISCUSSION TITLE
Table example

### DISCUSSION THREAD

**Discussion by pz-max** [pypsa-meets-africa/demand-creator#2] [General]
### Datasets (updated)

The following table contains datasets relevant to the demand creator tools. Two delineations of datasets are as follows:
- Target: Demand/Consumption
- Predictors: Wind, Humidity, GDP, Buildings, etc..

| Source |                 Data Name                |             Description              |       Regions     | Time Period |  Data format  | Access Details (Open/Proprietary/etc..) |
|----------|--------------------------|--------------------------|--------------------------|--------------------------|--------------------------|--------------------------|
| Post link here |    BIG DATA   | This is the BIG DATA and was created in YYYY. Time series data of demand or not demand?. |   REGIONS COVERED  |  YYYY-YYYY           | GIS/Time Series/Network/Graph/etc.. | OPEN/NOT OPEN/ ETC...|

**Comment by pmcwhannel** [comment]
> ### Datasets (updated)
> The following table contains datasets relevant to the demand creator tools. Datasets will
> 
> Source	Data Name	Description	Regions	Time Period	Additional Details	Access Details (Open/Proprietary/etc..)
> Post link here	BIG DATA	This is the BIG DATA and was created in YYYY. Time series data.	REGIONS COVERED	YYYY-YYYY	Dataset limitations/challenges perhaps?	OPEN/NOT OPEN/ ETC...

Please feel free to add any suggested columns for basic description for datasets relevant to the "demand-creator".

**Reply by pz-max** [reply]
Hey Pierre, We can extract relatively easily all the data that are mentioned here in 20x20km spatial resolution, 1-hour temporal resolution for over 30years: https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels?tab=form

Feel free to add some of it to the above list :)
